build/upload.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Consider adding `set -e` to this script, which will cause the entire script to stop and fail if any command within the script fails.